src/controller/EventController.js

```diff
@@ -0,0 +1,99 @@
+import InputView from '../InputView.js';
+import OutputView from '../OutputView.js';
+import EventService from '../service/EventService.js';
+import OrderService from '../service/OrderService.js';
+
+export default class EventController {
+  /**
+   * @private
+   */
+  #services = {
+    orderService: OrderService,
+    eventService: EventService,
+  };
+
+  constructor() {
+    OutputView.printWelcome();
+  }
+
+  /**
+   * @async
+   * @private
+   */
+  async start() {
+    let visitDayInput;
+
+    await this.#handleError(async () => {
+      visitDayInput = await InputView.readVisitDay();
+      this.#services.eventService.initialize(visitDayInput);
+    });
```

Comment:
```suggestion


    const visitDayInput = await this.#handleError(async () => {
      const visitDay = await InputView.readVisitDay();
      
      this.#services.eventService.initialize(visitDay);
      
      return visitDay
    });
```

p5: let 대신 const를 사용하도록 구현하는 방법이 있다면 사용해보는건 어떨까용? (에러 핸들러를 리턴 하도록 변경 필수!)